plugin/src/main/java/net/codingarea/challenges/plugin/challenges/implementation/goal/forcebattle/ForcePositionBattleGoal.java

```diff
@@ -0,0 +1,86 @@
+package net.codingarea.challenges.plugin.challenges.implementation.goal.forcebattle;
+
+import net.anweisen.utilities.common.config.Document;
+import net.codingarea.challenges.plugin.challenges.implementation.goal.forcebattle.targets.PositionTarget;
+import net.codingarea.challenges.plugin.challenges.type.abstraction.ForceBattleGoal;
+import net.codingarea.challenges.plugin.content.Message;
+import net.codingarea.challenges.plugin.management.scheduler.policy.TimerPolicy;
+import net.codingarea.challenges.plugin.management.scheduler.task.ScheduledTask;
+import net.codingarea.challenges.plugin.utils.item.ItemBuilder;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author sehrschlechtYT | https://github.com/sehrschlechtYT
+ * @since 2.2.3
+ */
+public class ForcePositionBattleGoal extends ForceBattleGoal<PositionTarget> {
+    public ForcePositionBattleGoal() {
+        super(Message.forName("menu-force-position-battle-goal-settings"));
+        registerSetting("radius", new NumberSubSetting(
+                () -> new ItemBuilder(Material.DIAMOND_BOOTS, Message.forName("item-force-position-battle-radius")),
+                value -> null,
+                value -> "§e" + (value * 100),
+                1,
+                100,
+                15
+        ));
+    }
+
+    @NotNull
+    @Override
+    public ItemBuilder createDisplayItem() {
+        return new ItemBuilder(Material.DIAMOND_BOOTS, Message.forName("item-force-position-battle-goal"));
+    }
+
+    @Override
+    protected PositionTarget[] getTargetsPossibleToFind() {
+        return new PositionTarget[0]; //Not used
```

Comment:
Just like here

Reply:
As the extreme force battle goal, the Force Position Battle Goal has its own `setRandomTarget` logic. Random position targets are created in a radius around the player's position. It's therefore not possible to list all possible position targets.